packages/design-system/src/components/ChoiceList/_ChoiceList.docs.scss

```diff
@@ -15,7 +15,7 @@ Style guide: components.choice
 
 A `ChoiceList` component can be used to render a radio button group, or checkbox group.
 
-By default the component determines the type of field for you, taking into account accessibility and usability best practices. So, you can pass in an array of `choices` and let it determine what type of field would be best for the user, or alternatively you can manually pass in the `type` prop.
+Pass in the `type` prop to specify the type of field to display for the user.
```

Comment:
Maybe this sentence can be combined with the one above, so something like: 

`A `ChoiceList` component can be used to render a radio button group, or checkbox group. Use the `type` prop to specify the type of field to display.`

Reply:
Thanks, that does read better.